README.md

```diff
@@ -31,6 +31,9 @@ Azure Toolkits for Java is an open-source project that helps Java developers eas
 * [Java Developer Center on Azure](https://docs.microsoft.com/en-us/java/azure/)
 * [Get Started for HDInsight](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-apache-spark-eclipse-tool-plugin)
 
+## Known issues
+Follow this [link](https://github.com/Microsoft/azure-tools-for-java/issues?q=is%3Aopen+is%3Aissue+label%3Aknown-issue) to track the known issues.
```

Comment:
How to handle the known issue we won't fix?

Reply:
Then I think it's better to show all  open and closed issues with the label. I'll update the PR.